docs/pages/enroll-resources/auto-discovery/kubernetes/google-cloud.mdx

```diff
@@ -454,8 +454,8 @@ value, `gke`.
 #### `discovery_service.gcp[0].project_ids`
 
 In your matcher, replace `myproject` with the ID of your Google Cloud project.
-The `project_ids` field must include at least one value, and it must not be the
-wildcard character (`*`).
+The `project_ids` field must include at least one value, and it must not mix the
```

Comment:
Does this mean that the value can either be a single wildcard character or a complete project ID?

Reply:
It means you can have ['*'], ['p1'], ['p1','p2'] but never ['*','p1']

Reply:
Thanks! I would include this example to make this clearer